server/src/instrument/6500/index.ts

```diff
@@ -206,7 +209,7 @@ namespace Buffer {
                 signatureExclusives: [
                     {
                         parameters: new Map([
-                            [ 0, bufferEnumUnit ],
+                            [ 0, bufferEnumUnitCustom ],
```

Comment:
This is the recommended approach. Some additional information:

Extracting a subset of enumerations into a unique array in this manner may not seem necessary when we only have a handful of `ExclusiveCompletionApiSpec` objects. However, it helps reduce label typos and missing entries since groups of related enumerations only need to be declared once.
This also applies to single `ExclusiveCompletionApiSpec` objects, as seen in the `Smu` namespaces of applicable instrument specification files.

I had a lot of trouble with copy-paste errors so I rewrote the instrument ApiSpec to more easily allow in-file (and possibly external-file) reuse. Since it was a conscious design decision, it may be worth including in the wiki somewhere.